README.md

```diff
@@ -38,6 +40,82 @@ To test the installation, run:
 zkpig version
 ```
 
+### Build from Source
+
+To build ZK-PIG from source, you'll need:
+- Go 1.21 or later
+- Git
+- Make
+
+1. Clone the repository:
+
+```sh
+git clone https://github.com/kkrt-labs/zk-pig.git
+cd zk-pig
+```
+
+2. Build the binary:
+
+```sh
+make build
```

Comment:
The command to build binaries locally is `make gorealeaser-snapshot` which generates binaries for all architectures under `/dist`